fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -275,6 +275,7 @@ def self.gem_path(gem_name)
 
     # Show/Hide loading indicator
     def self.show_loading_indicator(text = "🚀")
+      return if FastlaneCore::Env.truthy?("FASTLANE_DISABLE_ANIMATION")
       @require_fastlane_spinner = TTY::Spinner.new("[:spinner] #{text} ", format: :dots)
       @require_fastlane_spinner.auto_spin
     end
```

Comment:
one last nitpick, `self.hide_loading_indicator`  also needs to be worked on other wise it crashes as @require_fastlane_spinner  might not be initialized.

Reply:
okay, i think i did that just now.